components/Cards/src/Theming/MDCCard+MaterialTheming.m

```diff
@@ -14,12 +14,10 @@
 
 #import "MDCCard+MaterialTheming.h"
 
-#import "MaterialCards+ColorThemer.h"
-#import "MaterialCards+ShapeThemer.h"
-
 static const MDCShadowElevation kNormalElevation = 1;
 static const MDCShadowElevation kHighlightedElevation = 1;
 static const CGFloat kBorderWidth = 1;
+static const CGFloat kStrokeVariantBorderOpacity = (CGFloat)0.37;
```

Comment:
Tip: Nit: Should this be called `kOutlinedVariantBorderOpacity`? It looks like the style is called "Outlined Theme" below.

Reply:
Done.